README.md

```diff
@@ -1,5 +1,7 @@
 # Probot: Config
 
+> This project has been merged into [Probot](https://github.com/probot/probot) itself as part of the core `context.config` helper. Any future development should occur there.
```

Comment:
I think I'd make it more prominent. I'd put it as an h1 header on the very top of the repository so that nobody could ever overlook it. 

And instead of "should occur" I'd phrase it more decisive. E..g

> Any future development takes place there

Reply:
Pushed a new version.